sormas-ui/src/main/java/de/symeda/sormas/ui/visit/VisitGrid.java

```diff
@@ -59,6 +61,19 @@ public VisitGrid(VisitCriteria criteria) {
 		addEditColumn(
 			e -> ControllerProvider.getVisitController().editVisit(e.getUuid(), getCriteria().getContact(), getCriteria().getCaze(), r -> reload()));
 
+		removeColumn(VisitIndexDto.ORIGIN);
+		addComponentColumn(visitIndexDto -> {
+			VisitOrigin origin = visitIndexDto.getOrigin();
+
+			String displayText = origin.toString();
+
+			if (origin == VisitOrigin.USER) {
+				displayText += " " + visitIndexDto.getVisitUser().getShortCaption();
```

Comment:
I'm not pretty sure if visitIndexDto.getVisitUser() can be null, but if it can, than NullPointerException will be raised.
In my opinion is better to add as well a check for nonNull for visitIndexDto.getVisitUser().